elastic/datadog_checks/elastic/elastic.py

```diff
@@ -274,7 +274,7 @@ def _get_template_metrics(self, admin_forwarder, base_tags):
         try:
             template_resp = self._get_data(self._join_url('/_cat/templates?format=json', admin_forwarder))
         except requests.exceptions.RequestException as e:
-            self.log.error("Error reading templates info from servers (%s) - template metrics will be missing", e)
+            self.log.debug("Error reading templates info from servers (%s) - template metrics will be missing", e)
```

Comment:
I don't think we have one. My guess is here, since this endpoint is optional and the check can work without it since it's not vital, we do not absolutely need an `error` log. 

So I'd say: 

- Is it blocking the check? If yes then error, otherwise debug to avoid polluting the logs

What do you think?

Reply:
Makes sense